carbonmark-api/src/graphql/marketplace.gql

```diff
@@ -45,10 +45,14 @@ query getUserByWallet($wallet: Bytes) {
   }
 }
 
-query findProjects($search: String, $vintage: [BigInt!]) {
+query findProjects(
```

Comment:
Will need to add 

```ts
 expiresAfter: Type.Optional(
    Type.String({
      description:
        "Only return listings that expire after this timestamp (Unix seconds)",
      default: "Current system timestamp.",
      examples: ["1620000000"],
    })
  ),
```
to the schema for get and provide a default value in the handler (ts is complaining)

Also does this need to be on the /projects/[id] handler too?